src/api/cow/types.ts

```diff
@@ -135,4 +135,6 @@ export type PriceQuoteParams = Omit<FeeQuoteParams, 'sellToken' | 'buyToken'> &
 export type Options = {
   chainId?: ChainId
   isDevEnvironment?: boolean
+  reqOptions?: RequestInit
```

Comment:
Nitpick: can you name it something more clear?

I thought on first read it meant `required options`